src/components/dashboard/main-content/course-enrollments/data/hooks.js

```diff
@@ -143,16 +146,9 @@ export const useCourseUpgradeData = ({
       setIsLoading(true);
 
       try {
-        const containsContentResponse = await courseService.fetchEnterpriseCustomerContainsContent([courseRunKey]);
-        const { containsContentItems, catalogList } = camelCaseObject(containsContentResponse.data);
-
-        // Don't do anything if the course is not part of the enterprise's catalog
-        if (!containsContentItems) {
-          return;
-        }
-
         if (subscriptionLicense) {
           // get subscription license with extra information (i.e. discount type, discount value, subsidy checksum)
+          // TODO: Refactor to use react query
```

Comment:
If we were to nix the `license-subsidy` call altogether, I wonder if it'd make sense to move out of the `fetchData` function since it'd no longer be fetching anything here for the subscriptions case.